src/routes/transactions/__tests__/resources/incoming-transfers/e2e/erc20-expected-response.json

```diff
@@ -5,7 +5,7 @@
     {
       "type": "TRANSACTION",
       "transaction": {
-        "id": "transfer_i276abf83da6aabc61b429a1663b2bd647d5c9ff5594aa1d55ba49c70e99a27000,0,0",
```

Comment:
Safe address (second part of the id), was re-introduced because having the related `Safe` is needed to build the transaction details when retrieving a transaction by ID (see TransactionsService.getById`).